runtime/src/lib.rs

```diff
@@ -1040,6 +1040,29 @@ impl IsmpRouter for ModuleRouter {
     }
 }
 
+parameter_types! {
+    pub const Plonky2MaxPubsSize: u32 = 262_144;
+    pub const Plonky2MaxProofSize: u32 = 262_144;
+    pub const Plonky2MaxVkSize: u32 = 262_144;
```

Comment:
This might be too much indeed, I think we can set it to something like 50_000 and be happy with it